var/spack/repos/builtin/packages/easybuild/package.py

```diff
@@ -16,14 +16,12 @@ class Easybuild(PythonPackage):
     maintainers = ["boegel"]
 
     version("4.0.0", sha256="21bcc1048525ad6219667cc97a7421b5388068c670cabba356712e474896de40")
-    version("3.1.2", sha256="a966a08e41d3557d7fd3dc5322180c006d27281d9ecc1b1fce1b4518c32664d4")
 
-    depends_on("python@2.6:2.8", when="@:3", type=("build", "run"))
     depends_on("python@2.6:2.8,3.5:", when="@4:", type=("build", "run"))
```

Comment:
@adamjstewart Should this be changed to `3.5:`? Likewise in `py-easybuild-*`

Reply:
Technically, it supports all versions of Python that Spack supports, so we could just get rid of all version specifications. There are thousands of packages where this is true, so I haven't bothered to update all of them.